packages/agoric-cli/src/publish.js

```diff
@@ -198,7 +196,9 @@ export const makeHttpBundlePublisher = ({ jsonHttpCall, getAccessToken }) => {
       'object',
       X`Expected JSON object response body, got ${response}`,
     );
-    assert(response, X`Expected non-null response body, got ${response}`);
+    if (!response) {
+      throw Fail`Expected non-null response body, got ${response}`;
+    }
```

Comment:
why not the `||` style?
```suggestion
    response || Fail`Expected non-null response body, got ${response}`;
```

Is there a guideline for when to use `||` or `if` ?

Reply:
Thanks for filing that!

My interpretation is that, with this bug filed, there's nothing more I need to do here. So resolving.